metrics-api/src/main/java/software/amazon/swage/metrics/ContextData.java

```diff
@@ -18,33 +18,33 @@
 import software.amazon.swage.collection.TypedMap;
 
 /**
- * Builder for data carried with the context under which a metric is reported.
+ * Builder for dimensions carried with the context under which a metric is reported.
  *
  * <p>
- * Roughly equivalent to a request context, this is the metadata associated with
+ * Roughly equivalent to a request context, this is the dimensions associated with
```

Comment:
Any particular reason to prefer 'dimensions' over 'metadata'?  I prefer the latter, as 'dimensions' has pre-existing meaning and connotations in some existing metrics systems.  While the meaning here is _almost_ the same, better not to conflate the two.

Reply:
Not enough words in the language. Yeah, agree "dimensions" might be confusing especially if we switch to using JSR 363 UoM which has a Dimension concept.

"metadata" seemed a bit generic to me. How about:
- "attributes"
- "qualifiers"
- "parameters"